buildenv/jenkins/variables/defaults.yml

```diff
@@ -298,6 +351,63 @@ s390x_linux_xl:
     13:
       ? special.system
 #========================================#
+# Linux S390 64bits Compressed Pointers /w JITSERVER
+# Note: boot_jdk 8 must use an Adopt JDK8 build rather than an
+# IBM 7 for the bootJDK or compiling corba will fail to find Object.
+#========================================#
+s390x_linux_jit:
+  boot_jdk:
+    8: '/usr/lib/jvm/adoptojdk-java-s390x-80'
+    11: '/usr/lib/jvm/adoptojdk-java-11'
+    12: '/usr/lib/jvm/adoptojdk-java-11'
+    13: '/usr/lib/jvm/adoptojdk-java-12'
+    next: '/usr/lib/jvm/adoptojdk-java-13'
+  release:
+    8: 'linux-s390x-normal-server-release'
+    11: 'linux-s390x-normal-server-release'
+    12: 'linux-s390x-server-release'
+    13: 'linux-s390x-server-release'
+    next: 'linux-s390x-server-release'
+  freemarker: '/home/jenkins/freemarker.jar'
+  openjdk_reference_repo: '/home/jenkins/openjdk_cache'
+  node_labels:
+    build:
+      8: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+      11: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+      12: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+      13: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+      next: 'ci.role.build && hw.arch.s390x && sw.os.ubuntu'
+  build_env:
+    vars:
+      8: 'CC=gcc-7 CXX=g++-7'
+      11: 'CC=gcc-7 CXX=g++-7'
+      12: 'CC=gcc-7 CXX=g++-7'
+      13: 'CC=gcc-7 CXX=g++-7'
+      next: 'CC=gcc-7 CXX=g++-7'
+  extra_getsource_options:
+    8: '--openssl-version=1.1.1d'
+    11: '--openssl-version=1.1.1d'
+    12: '--openssl-version=1.1.1d'
+    13: '--openssl-version=1.1.1d'
+    next: '--openssl-version=1.1.1d'
+  extra_configure_options:
+    8: '--with-openssl=fetched --enable-jitserver=yes'
+    11: '--with-openssl=fetched --enable-jitserver=yes'
+    12: '--with-openssl=fetched --enable-jitserver=yes'
+    13: '--with-openssl=fetched --enable-jitserver=yes'
+    next: '--with-openssl=fetched --enable-jitserver=yes'
+  excluded_tests:
+    11:
```

Comment:
Although this is JitServer and not Java, please excluded 8 here as well, just in case. We don't have the machines to add any more special.system testing in the nightly builds.